assets/js/filter.js

```diff
@@ -209,7 +298,7 @@ $(function() {
             } else {
                 input.off('hide.bs.popover');
                 input.popover("destroy")
-                    .removeAttr("class title data-original-title");
+                    .removeAttr("title data-original-title");
```

Comment:
Not sure why `class` was originally in the list of attributes to be removed. It caused the input box to lose its styling after the popover was closed, so I removed it from the list.